store/postgres/src/deployment_store.rs

```diff
@@ -1352,4 +1347,12 @@ impl DeploymentStore {
                   "shard" => self.pool.shard.as_str())
         });
     }
+
+    pub(crate) fn health(
+        &self,
+        id: &DeploymentHash,
+    ) -> Result<deployment::SubgraphHealth, StoreError> {
+        let conn = self.get_conn()?;
+        deployment::health(&conn, id)
```

Comment:
This is a blocking function called from an async context, we should make it async by using `with_conn`.

Reply:
Code looks fine, thanks for addressing this!